app/Http/Controllers/TransactionMaterialReceivingItemController.php

```
@@ -227,4 +128,23 @@ public function reject(TransactionMaterialReceivingItem $resource, TransactionMa
+    {
+        $mrr = $resource->transactionMaterialReceiving;
+        $checks = [
+            [$mrr->supplier_id !== null, 'Supplier has not been updated yet.'],
+            [$mrr->reference !== null, 'Reference has not been updated yet.'],
+            [$mrr->terms_of_payment !== null, 'Terms of payment have not been updated yet.'],
+            [$mrr->particulars !== null, 'Particulars have not been updated yet.'],
+            [$resource->specification !== null, 'Item specification has not been updated yet.'],
+            [$resource->actual_brand_purchase !== null, 'Actual brand purchased has not been updated yet.'],
+            [$resource->unit_price !== null, 'Unit price has not been updated yet.'],
+        ];
+        foreach ($checks as [$ok, $message]) {
```

PHPMD (2.15.0)

Avoid variables with short names like $ok. Configured minimum length is 3. (Naming Rules)

(ShortVariable)